spec/support/acts_as_tenant.rb

```diff
@@ -59,9 +59,22 @@ def with_tenant(tenant, &proc)
   end
 end
 
+module ActsAsTenant::TestExampleHelpers
+  module FeatureHelpers
+    [:visit, :click_button].each do |method|
+      define_method(method) do |*args|
+        ActsAsTenant.with_tenant(nil) do
```

Comment:
Because Capybara is supposed to set the correct host, then the Rack stack is supposed to process the request. The current tenant is supposed to be set inside our controller code.

Before the request: we set tenant
During the request, we unset the tenant. The controller needs to re-deduce the tenant
After the request, the tenant is unset by Rack. We reset the tenant.